core/src/main/java/org/testcontainers/utility/RegistryAuthLocator.java

```diff
@@ -79,12 +82,6 @@ static void setInstance(RegistryAuthLocator overrideInstance) {
      */
     public AuthConfig lookupAuthConfig(DockerImageName dockerImageName, AuthConfig defaultAuthConfig) {
 
-        if (SystemUtils.IS_OS_WINDOWS) {
-            log.debug("RegistryAuthLocator is not supported on Windows. Please help test or improve it and update " +
-                "https://github.com/testcontainers/testcontainers-java/issues/756");
-            return defaultAuthConfig;
-        }
```

Comment:
😍 I already love this PR!

Will give this a proper review and try later today, but thank you for taking this on!